data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+			choice
+				`	"Sounds easy enough. Where should I start?"`
+				`	"Whether they go undercover or not, this seems a bit risky for me. Sorry, you'll have to find someone else."`
+					decline
+			`	"Your choice, that is," Iekie responds. "Known to these outlaws, many of our methods are. Easily predicted, our ordering would be."`
+			`	Some of the others explain that you will have to visit the following worlds they have deemed suspicious: Shifting Sand, in Saryd space, Into White, Fourth Shadow and Remote Blue, in Kimek space, and Mebla's Portion and Stronghold of Flugbu, in Arach space.`
+			`	They wish the four of you luck. You're escorted back to the spaceport, after which you show the 'tourists' to their bunks in your ship.`
+				accept
+	on complete
+		"assisted heliarch" ++
+		log "Assisted the Heliarch in investigating several worlds around Coalition space, bringing their agents to each world to look for any signs of criminal activity."
+		payment 900000
+		conversation
+			`The Heliarch consul, Iekie, greets you and the agents when you return, handing you your payment of <payment>.`
+			`	"Unearthed some useful information, we have," Tugroob tells her, as he delivers the agents' reports. "Essential to the mission's success, Captain <last>'s cooperativeness was."`
+			`	"No doubt of that, I have. To the report, looking forward I am."`
+			branch evidence1
+				"resistance evidence" == 1
+			branch evidence2
+				"resistance evidence" == 2
+			branch evidence3
+				"resistance evidence" == 3
+			label finalthanks
+			`	They thank you once again for helping, and move to the Heliarch-only parts of the ringworld.`
+				accept
+			label evidence1
+			`	The agents then get closer to her before Iplora whispers something, handing her another report.`
+			`	Iekie turns to you once more, saying, "Quite a find, this is. Done well, you have, Captain."`
+				goto finalthanks
+			label evidence2
+			`	The agents then get closer to her before Iplora whispers something, handing her two separate reports.`
+			`	Iekie turns to you once more, saying, "Most useful to our investigation, these findings will prove. Excellent work, Captain."`
+				goto finalthanks
+			label evidence3
+			`	The agents then get closer to her before Iplora whispers something, handing her three separate reports.`
+			`	Iekie turns to you once more, saying, "Enviable, your prowess at investigation is, <last>. Outstanding work, Captain."`
+				goto finalthanks
+
+mission "HI2-Bonus 1"
+	invisible
+	landing
+	to offer
+		has "Heliarch Investigation 2: done"
+		"resistance evidence" == 1
+	source
+		government "Coalition"
+	on offer
+		payment 300000
+		conversation
+			apply
+				clear "resistance evidence"
+			`You receive a message from the Heliarch Consul Iekie as you go in for a landing.`
+			`	"Captain <last>! Analyzed your findings, we have. Brought light to some stolen equipment, your aid has, and for that, grateful we are. Sending a monetary bonus as thanks, I am."`
```

Comment:
The theft of Heliarch equipment is made possible/facilitated by the combined efforts of:

- The Quarg, who relay to the Lunarium intelligence they get via stalking every Heliarch decision made in the rings,
- House Bebliss, the most advanced group in the Coalition in terms of understanding hyperlane communications, and the group responsible for mantaining such a network,
- House Idriss, the Arach House specialized on all things computers and electronics, and who are more than capable of jeopardizing most if not all Heliarch surveillance systems on a small area,
- Elirom, a former Heliarch Arbiter (second highest rank) who has a very good understanding of their modus operandi, and who can thus advise the Lunarium on the best way to go about their operations.

By comparison, the "minor rare accident" from your Friend of Arthropods PR is made possible/facilitated by:

- 


I would say that it's pretty consistent that a society that can prepare enough for the weather to the extent that they're essentially immune to such accidents, can still be susceptible to various major factions and former high ranking members working against them, so much so that equipment can be stolen and hidden away from such a society's government.

The Lunarium is getting aid from a whole lot of people. The weather accident in your PR is just that, an accident, pure chance. They're not at all comparable.
One of the alternatives I provided **is exactly making it not an accident, but instead product of drastic actions taken by the Heliarchs**.
I won't discuss that PR any more here, and I'd appreciate if you kept the discussion of your PR to your PR.